sputnikdao2/src/proposals.rs

```diff
@@ -410,6 +410,9 @@ impl Contract {
                         ValidAccountId::try_from(token_id.clone()).is_ok(),
                         "ERR_TOKEN_ID_INVALID"
                     );
+                } else {
+                    // Don't create a Transfer proposal for Ⓝ that cannot be paid.
+                    assert!(env::account_balance() >= amount.0, "ERR_NOT_ENOUGH_BALANCE")
```

Comment:
The full balance is not transferrable, because of storage and also potentially there are other money which are needed for other usages (like already committed in bounties). 

It's unclear how to facilitate correct check here for me actually. I was originally thinking of tracking "available" balance and subtract every time there is "commitment" to pay out but it's unclear when to commit the funds.